lib/index.js

```diff
@@ -14,7 +14,7 @@ const {
 } = require("./config-array-factory");
 
 const { CascadingConfigArrayFactory } = require("./cascading-config-array-factory");
-const { ModuleResolver } = require("./shared/relative-module-resolver");
+const { resolve } = require("./shared/relative-module-resolver");
```

Comment:
Can we do `const ModuleResolver = require(...)` and then export `ModuleResolver` to just fix and keep the intended API?